DQMOffline/Trigger/python/TopMonitoring_Client_cff.py

```diff
@@ -1,4 +1,174 @@
 import FWCore.ParameterSet.Config as cms
 
+topEfficiency_elejets = cms.EDAnalyzer("DQMGenericClient",
+    subDirs        = cms.untracked.vstring("HLT/TopHLTOffline/TopMonitor/EleJet/*"),
+    verbose        = cms.untracked.uint32(0), # Set to 2 for all messages
+    resolution     = cms.vstring(),
+    efficiency     = cms.vstring(
+        "effic_metME       'efficiency vs MET; MET [GeV]; efficiency' metME_numerator       metME_denominator",
```

Comment:
Thanks Sam. How can I get relevant updates for that? I would like to keep testing my code with my current setup